process-engine/bpmn-studio
# ✨ Add File Association for the Electron Application

## What did you change?

This PR adds file association for files with a `.bpmn` extension.

Currently it only opens the bpmn-studio app and 
safes the file path and file content into an object when double clicking on a `.bpmn` file.

Closes #247 

the https://github.com/process-engine/bpmn-studio/pull/243 pull request should merged before this, because this PR includes the changes from #243 

Bezüglich des genutzten NotificationServices ist hier noch ein relevantes Issue:
https://github.com/process-engine/bpmn-studio/issues/255

## How can others test the changes?

Info: Das process_engine_http package muss im `skeleton/process-engine-server` in der package.json auf `^1.1.0` stehen.

- Checkout the branch
- `npm install`
- `npm run build`
- `npm run electron-build`
- Install andstart BPMN-Studio with the generated .dmg file 
- Close it and double click on a `bpmn` file

The BPMN-Studio app should start now.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
